domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/MissingHeaderNotice.java

```diff
@@ -25,8 +25,8 @@
 
     public MissingHeaderNotice(final String filename, final String missingHeaderName) {
         super(filename, E_001,
-                "Field name not defined in the GTFS specification",
-                "File `" + filename + " is missing required field name: `" + missingHeaderName + "`",
+                "Missing required field",
+                "File `" + filename + " is missing required field: `" + missingHeaderName + "`",
```

Comment:
You could remove the leading `"File " + ` as well to make this more concise - it should be obvious to anyone running the tool what *.txt is. Also note that currently the filename is missing the trailing tick, if you decide to keep as-is.